readyplayerme/rpm-unity-sdk-avatar-loader
# General cleanup and refactor

## [TICKETID](https://ready-player-me.atlassian.net/browse/TICKETID)

## Description

-   There was no task for this but I just did a general pass and cleanup of avatar loader. 
Some 




## Changes

#### Updated

-   general cleanup and refactoring of various classes



## How to Test

-   Check that tests pass and open and run a few scenes, make sure there are no errors



## Checklist


## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.
